accounts/models.py

```diff
@@ -41,6 +40,27 @@ class User(AbstractUser):
     USERNAME_FIELD = 'email'
     REQUIRED_FIELDS = []
 
+    def __str__(self):
+        return self.email
 
-def __str__(self):
-    return self.email
+
+class Garden(models.Model):
+    userId = models.ForeignKey(
+        User, on_delete=models.CASCADE)  # on cascade delete
+    description = models.TextField(blank=True, null=True)
+    title = models.CharField(max_length=100)
+    address = models.TextField()  # todo address
+    # todo specific Geo models ? https://pypi.org/project/django-address/
+    zipcode = models.CharField(max_length=5, validators=[])
```

Comment:
Pas la peine de mettre le validateur si c'est vide pour l'instant (mais on peut mettre un #todo
```suggestion
    zipcode = models.CharField(max_length=5) # TODO: check best practices
```